Tiltfile

```diff
@@ -169,6 +169,15 @@ def capz():
 
     k8s_yaml(blob(yaml))
 
+def create_identity_secret():
+    #create secret for identity password
+    os.putenv('CLUSTER_IDENTITY_SECRET_NAME', 'cluster-identity-secret')
```

Comment:
will we need to add the namespace variable to tilt-settings (if so needs to be added to tilt docs), or could we just default it / set it here?

Reply:
its defaulting to `default` here currently, same as the script `hack/create-identity-secret.sh` 

for the e2e tests, its in the same namespace as the cluster

not sure what we need to change?

Reply:
it looks like the templates expect `AZURE_CLUSTER_IDENTITY_SECRET_NAMESPACE` to be set, should that be set here? Same question for `CLUSTER_IDENTITY_NAME`.

Basically, I'm wondering if I pull the changes from this PR and run `make tilt-up` and then click on one of the flavors to deploy it, will everything work as it did before? or are there any additional vars that need to be defined in my tilt-settings.json file now? It would be nice if everything kept working as before.

Reply:
I think it should work the same, I added the setting of the namespace and identity name in the tiltfile similar to the script

Reply:
> @nader-ziada tried out tilt locally, found two issues:
> 
> * if you run tilt up a second time it fails with `error: failed to create secret secrets "cluster-identity-secret" already exists`: we should delete the secret and recreate it every tilt up so it's idempotent (same thing we do with configmaps)
> * the secret env var is getting printed in tilt logs, any way we can avoid that?

where does it delete configmaps? can't find that

Reply:
here: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/Tiltfile#L174